src/main/java/io/split/android/client/localhost/LocalhostSplitFactory.java

```diff
@@ -82,6 +83,11 @@ public SplitClient client() {
         return mClient;
     }
 
+    @Override
+    public SplitClient client(Key key) {
+        return mClient;
```

Comment:
@gthea , check if we need to create different client instances in localhost mode too, so that each shared client emits its own sdk events (SDK_READY, etc).

Reply:
For sure, Localhost mode will be updated in a future PR.